sdk/iOS/src/MSTable.h

```diff
@@ -162,12 +162,12 @@ extern NSString *const MSSystemColumnDeleted;
 /// configured and then executed to retrieve items from the table. An *MSQuery*
 /// instance provides more flexibilty when querying a table than the table
 /// read* methods.
--(MSQuery *)query;
+-(nonnull MSQuery *)query;
 
 /// Returns an *MSQuery* instance associated with the table that uses
 /// the given predicate. An *MSQuery* instance provides more flexibilty when
 /// querying a table than the table read* methods.
--(MSQuery *)queryWithPredicate:(NSPredicate *)predicate;
+-(nonnull MSQuery *)queryWithPredicate:(nullable NSPredicate *)predicate;
```

Comment:
I had this as nullable as in the other constructor `-[MSQuery initWithTable:]` the predicate parameter is passed as nil. Hence assumed the parameter was nullable further up the tree. But in the context of this one call, you would just use the `-[MSTable query]` method if you had a nil predicate.